.bazelrc

```diff
@@ -1,5 +1 @@
-build --workspace_status_command="python tools/workplace_status.py"
-
-# Flags used during releases
-build:release --stamp
-test:release --stamp
+build --workspace_status_command="python tools/workplace_status.py" --stamp
```

Comment:
Can you restore the config mode of `release`? Forcing stamp on all the time causes cache misses when I checked this out locally for testing.

Reply:
Alright, done!